egui/src/containers/frame.rs

```diff
@@ -52,7 +89,7 @@ impl Frame {
 
     pub fn window(style: &Style) -> Self {
         Self {
-            margin: style.spacing.window_padding,
+            margin: style.spacing.window_padding.into(),
```

Comment:
`window_padding` should perhaps be renamed `window_margin` and be made into a `Margin` too?